pkg/commands/option/artifact.go

```diff
@@ -37,16 +37,11 @@ func NewArtifactOption(c *cli.Context) ArtifactOption {
 
 // Init initialize the CLI context for artifact scanning
 func (c *ArtifactOption) Init(ctx *cli.Context, logger *zap.SugaredLogger) (err error) {
-	// kubernetes subcommand doesn't require any argument
-	if ctx.Command.Name == "kubernetes" {
-		return nil
-	}
-
 	if c.Input == "" && ctx.Args().Len() == 0 {
 		logger.Debug(`trivy requires at least 1 argument or --input option`)
 		_ = cli.ShowSubcommandHelp(ctx) // nolint: errcheck
 		os.Exit(0)
-	} else if ctx.Args().Len() > 1 {
+	} else if ctx.Args().Len() > 1 && ctx.Command.Name != "kubernetes" {
```

Comment:
Does k8s subcommand accept multiple commands?

Reply:
The idea is supporting resources scanning like kubectl:

eg: 

```
trivy k8s pod pi
trivy k8s deploy orion
...
```

Reply:
Thanks. It makes sense.